lib/Client.php

```diff
@@ -49,7 +49,7 @@ public function getDefaultTimeout()
 
     /**
      * @param RequestInterface $apiRequest
-     * @return \stdClass
+     * @return mixed
```

Comment:
By default, `json_encode` provides a mixed output due to it's second parameter `$assoc`. As we keep `$assoc` with it's default value `false`, the output for this function should always provide a `stdClass`, so could you please keep the `stdClass` reference?